src/time-polyfill/ReactTimeInputPolyfill.js

```diff
@@ -109,7 +109,7 @@ export default class TimeInputPolyfill extends React.Component {
 		}
 	}
 	componentWillUnmount() {
-		if (this.$input.current.form) {
+		if (this.$input.current && this.$input.current.form) {
```

Comment:
I'm wondering if we can use optional chaining instead 🤔 

I'm not sure if my build/release process will transform it for IE though.

This is probably the safer option I guess...

Reply:
Optional chaining will work, but depends on babel config/setup so went with this for now as it was safer (especially as Issue had been open since March)